doc/reference/make.md

```diff
@@ -9,36 +9,184 @@ See also the [guide](/guide/build.md).
+    This prefix will affect how your modules are named and where they will be installed
+    within your *libdir*. Thus, a prefix of `legs` means that a file `"foo/bar.ss"`
+    will be compiled into a module `legs/foo/bar` that you can import with `(import :legs/foo/bar)`
+    and that will be installed under your *libdir* as a bunch of files starting with the prefix
+    `legs/foo/bar`. The default, `#f`, means that no prefix will be prepended.
+
+  - `libdir:` specifies the directory under which compiled library files will be installed.
+    If left unspecified, the default `#f` designates that
+    the content of your `GERBIL_PATH` environment variable be used if it is defined,
+    or else your `~/.gerbil/lib/` directory.
+
+  - `bindir:` specifies the directory under which compiled executable files will be installed.
+    If left unspecified, the default `#f` designates that
+    your `~/.gerbil/bin/` directory shall be used.
+
+  - `force:` specifies a boolean that if true forces the recompilation
+    of every target in the current build. If for some reason a subtle change
+    causes the build system to incorrectly believe some targets are up-to-date
+    when they are not, you can use `force: #t` to force a complete rebuild.
+    Note that usually, this means that you failed to specify a dependency
+    (for instance, by using `(include ...)` and not specifying the file
+    in a `dep:` argument to a `(gxc: ...)` target).
+    It may also be a bug in Gerbil, in which case, please report it.
+
+  - `optimize:` specifies a boolean that if true tells Gerbil to optimize the code some more,
+    which should result in faster code, at the cost of compilation taking more time.
+    The default is `#t`.
+
+  - `debug:` specifies one of `#f`, `env` or `src`, and controls how much debugging information
+    Gerbil will include in the compiled file. The default is `env`.
+    With `#f` Gerbil drops all debug information;
+    with `env` it keeps a lot of information about the variables;
+    with `src` it additionally keeps around the expanded source code in the compiled output.
+    Each level makes debugging easier at the cost of larger output, and larger compilation time.
+    In production, you probably want `#f` or maybe `env` if operators can sometimes debug there;
+    for development, you probably want `src` for libraries you don't compile often,
+    and `env` for code you compile over and over.
+    Be sure to properly quote the symbol when passing it as argument to a function, as in
+    `(make build-spec debug: 'src)`.
+
+  - `static:` specifies a boolean that if true tells Gerbil to compile objects for inclusion
+    in a static binary in addition to shared objects that can be dynamically loaded at the REPL.
+    If you know you'll never generate a static binary from your project you can set it to `#f`,
+    but if your project is includes a static binary, or is a library that might be used
```

Comment:
extraneous "is": "if our project ~is~ includes ..."

Reply:
thanks